tests/test_settings.py

```diff
@@ -12,6 +12,10 @@
 assert reset_config
 
 
+# Set path to this directory
+here = os.path.abspath(os.path.dirname(__file__))
```

Comment:
Maybe change this to 
```suggestion
temp_dir = os.path.abspath(os.path.dirname(__file__)) if platform.system() == "Windows" else "/tmp/"
```
and use it below?

Reply:
I will test it again as soon as possible 👍